gslib/tests/test_json.py

```diff
@@ -0,0 +1,31 @@
+from gslib.tests import testcase
```

Comment:
Q: Any reason this file isn't named `test_<name-of-module>.py`?  I'd expect this to be named `test_json_six.py` if you're testing the `json_six` module.

Reply:
Yes, that's probably what it should be named. But now that I think about it, I'm pretty sure we can actually delete this test and the json_six file. I think my original intention was to use it throughout the program, but with the six.ensure methods, I think this may be irrelevant.